bip-0158.mediawiki

```diff
@@ -80,9 +83,13 @@ The following sections describe each step in greater detail.
 
 The first step in the filter construction is hashing the variable-sized raw
 items in the set to the range <code>[0, F)</code>, where <code>F = N *
-2^P</code>. Set membership queries against the hash outputs will have a false
-positive rate of <code>2^(-P)</code>. To avoid integer overflow, the number of
-items <code>N</code> MUST be <2^32 and <code>P</code> MUST be <=32.
+M</code>. Customarily, <code>M</code> is set to <code>2^P</code>. However, if
```

Comment:
Drop the sentence `Customarily, M is set to 2^P`. It's not adding anything and just raises questions. You can simply say that parameters are selected to maximize compression efficiency and link to the analysis.

Reply:
IMO it does add something as if one if familiar with the regular selection of parameters, why might wonder why we didn't just set M to 2^P.

Reply:
If someone is really curious, that's what the link is for. But I guess leave it if you feel strongly.